docker/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM golang:1.10-alpine AS builder
 ARG SVC_NAME
+ARG GOARCH
+ARG GOARM
```

Comment:
Same here

Reply:
It is used in `&& make $SVC_NAME \` which needs this arguments.

Reply:
Please set ARGs as:
1. ARCH
2. VER

Reply:
This is not new variables, they are already being used and in current docs we have them like `GOARCH` and `GOARM`: https://mainflux.readthedocs.io/en/latest/dev-guide/#cross-compiling-for-arm
If I change it, I will need to change that also. do you still want this change?